indexing-service/src/test/java/org/apache/druid/indexing/common/task/batch/parallel/SinglePhaseParallelIndexingTest.java

```diff
@@ -197,62 +182,52 @@ private void testRunAndOverwrite(@Nullable Interval inputInterval, Granularity s
   }
 
   @Test
-  public void testWithoutInterval() throws Exception
+  public void testWithoutInterval()
   {
     testRunAndOverwrite(null, Granularities.DAY);
   }
 
   @Test()
-  public void testRunInParallel() throws Exception
+  public void testRunInParallel()
   {
     // Ingest all data.
-    testRunAndOverwrite(Intervals.of("2017/2018"), Granularities.DAY);
+    testRunAndOverwrite(Intervals.of("2017-12/P1M"), Granularities.DAY);
```

Comment:
If the interval had been a named constant for the class, then updating it for all of the tests would have been easier. Perhaps it's worth creating the named constant now.

Reply:
Hmm, since the test data is generated in each class, I think it's better to have the constant separately in each class as well. Added constants.